nestkernel/conn_builder.cpp

```diff
@@ -85,11 +85,13 @@ nest::ConnBuilder::ConnBuilder( Network& net,
   // be possible to set the delay without the weight.
   default_weight_ = !syn_spec->known( names::weight );
 
+  default_delay_ = !syn_spec->known( names::delay );
```

Comment:
As far as I remember these values are defined and needed outside of the structural plasticity framework. We might discuss this point on another thread.